_posts/2021-03-14-curating/curating.Rmd

```diff
@@ -0,0 +1,329 @@
+    self_contained: false
+draft: false
+creative_commons: CC BY
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+```{r}
+folder <- "2021-03-14-curating"
+```
+
+```{r}
+library(tidyverse)
+library(ggtext)
+library(ggiraph)
+library(tweetrmd)
+```
+
+
+```{r, fig.cap="Photo by Arno Smit on Unsplash (https://unsplash.com/photos/iI72r3gSwWY)", fig.alt="Image of open door to porch with inviting sitting chair, surrounded by plants."}
+knitr::include_graphics("img/arno-smit-iI72r3gSwWY-unsplash.jpg")
+```
+
+# TL; DR
+
+In February 2021 I tweeted to a daunting >20k followers by curating for [`@WeAreLadies`](https://twitter.com/WeAreRLadies){target="_blank"} on Twitter. This was great opportunity to share knowledge, connect with others, and learn something in return, ultimately cultivating new connections and collaborations. From strategy to fruition, I hope this post helps you confidently enroll as a curator!
+
+# About `@WeAreRLadies`
+
+The `@WeAreRLadies` rotating Twitter curator exists to "encourage and maintain Twitter engagement within the R-Ladies community", and to "spotlight female and minority genders" working with R. R-Ladies has a [comprehensive guide](https://guide.rladies.org/rocur/about/){target="_blank"} describing the program, procedures and protocols for the week, and tips for successful curation. 
+
+# Overcoming imposter syndrome
+
+You may be hesitant to sign up as a curator due to [imposter syndrome](https://caitlinhudon.com/2018/01/19/imposter-syndrome-in-data-science/){target="_blank"} - I certainly was. I was on Twitter for three years before I gathered the courage. However, you do not need to know everything about R in order to be a successful `@WeAreRLadies` curator - that is impossible! In fact,
+
+
+```{r}
+tweetrmd::include_tweet("https://twitter.com/WeAreRLadies/status/1361660625718571010")
+```
+
+Every R user, new or experienced, has a valuable perspective to share. I was particularly  impressed when [Danielle Brantley](@daniebrant){target="_blank"} excellently curated in September 2020, after being an R user for one year!
+
+My personal strategy for combating imposter syndrome is to prepare. For my curating week, my preparation involved reflecting on past curators and creating some content in advance. I hope this post helps you to prepare and motivates you to sign up. `r emo::ji("wink")`
+
+# Date selection
+
+You can view the [schedule](https://docs.google.com/spreadsheets/d/1pmphnR9EinGuAKbVqGzMIt6cWYmHNbeFVzaKJB59JdY/edit#gid=1429703967){target="_blank"} of upcoming curators to identify available dates; records of previous curators are also maintained here.  
+
+Being a curator will be time intensive, so be kind to yourself. Choose dates when you will have time to invest and a flexible work schedule. I chose Feb 15-20 
+because I hoped by then I would be recovered from an intense Q4 work cycle;
+additionally, Feb 15 (President's Day) was a company holiday. Additionally, you may
```

Comment:
"additionally" appears twice on this line.

Reply:
good catch!